### DISCUSSION TITLE
More colorful Java code

### DISCUSSION THREAD

**Discussion by kaikeSantos** [tyron12233/CodeAssist#190] [Ideas]
Example: String a = 

LinearLayout a = 

Int a = 1;

View 

(LinearLayout)

...

Variables with colors

**Comment by tyron12233** [comment]
Thanks for the suggestion. Semantic highlighting is already planned and will be implemented in the future versions. Next time, use the discussions tab to submit feature requests as this is `strictly` for issues only. I will be converting this to a discussion.
